### ISSUE TITLE
Inconsistent Openlayers libraries

### ISSUE THREAD

**Issue by Gary-van-Breda** [Indicia-Team/media#21]
I know OL2 is very old hat at this point, but...
The contents of the Openlayers JS variants are inconsistent: e.g. the debug and min versions contain OpenLayers.Layer.TileCache, but the normal one does not.

**Comment by johnvanbreda** [comment]
Agreed but I think that's always been the way. The main version was built using the OL deployment builder with only the standard classes used by Indicia at the time. So if you want a full version you need to use another file.
I'm not sure this is worth revisiting at this point?

**Comment by Gary-van-Breda** [comment]
OpenLayers.Layer.TileCache is used by jquery.indiciaMapPanel.js, and the helper_base.php points to OpenLayers.js (without the OpenLayers.Layer.TileCache): perhaps helper_base should use OpenLayers-min instead?
